onboarding_protocol.md

```diff
@@ -5,6 +5,13 @@ The things we need to do to onboard a new person onto the team.
 - [ ] Create Google Apps account
 - [ ] Create IRCCloud account
   - [ ] Download IRCCloud mobile client
+- [ ] Add to calendars
+  - [ ] If they go to https://calendar.google.com they can add the calendar of
+    teammates by typing in their email (ex. `zaphod@hackedu.us`) into the
+    text box under "Other calendars" on the far left.
+  - [ ] Add the shared team calendar by pasting
+    `hackedu.us_l9sps0raf14hmn721076a6f9ms@group.calendar.google.com` into that
+    same text box
 - [ ] Add to GitHub organization
```

Comment:
Thoughts on adding the IRC chat bot?

```
steward notify me on mentions
```

I think that by doing this, it will allow a new team member to only have to remember to use 1 system, not 2.
They JUST need to get used to answering IRC and do not have to worry about checking GitHub because IRC will tell them about new GitHub issues.